csrc/id_model/indexing_traversal.cpp

```diff
@@ -65,18 +86,26 @@ std::optional<IndexingTraversal::ExprPath> IndexingTraversal::
       /*build_graphs=*/false);
 
   // Gather all resize exprs for each of the inputs and outputs
-  std::unordered_map<Val*, std::vector<Resize*>> tv_resize_map;
-  for (auto inp : ir_utils::filterByType<TensorView>(expr->inputs())) {
-    for (auto expr : inp->domain()->allExprs()) {
+  std::unordered_map<TensorView*, std::vector<Resize*>> tv_resize_map;
+  for (auto inp : expr->inputs()) {
```

Comment:
This was a trivial bug that I assumed `expr->inputs()` are still `TensorView` and not `TensorIndex`, which is not true for predicating vectorized ops.